common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -151,8 +151,7 @@ public void registerExecutor(
 
   /**
    * Obtains a FileSegmentManagedBuffer from a shuffle block id. We expect the blockId has the
-   * format "shuffle_ShuffleId_MapId_ReduceId" (from ShuffleBlockId), and additionally make
-   * assumptions about how the hash and sort based shuffles store their data.
+   * format "shuffle_ShuffleId_MapId_ReduceId" (from ShuffleBlockId).
    */
   public ManagedBuffer getBlockData(String appId, String execId, String blockId) {
```

Comment:
Is this method used anywhere else? I only see `ExternalShuffleBlockHandler` using this class, and it now uses the new method. If only unit tests use this, then remove this method and fix the unit tests.